src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/Example1.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="CommentsIndentation"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+// xdoc section -- start
+public class Example1 {
+  void testMethod() {
+    /*
+     * it is Ok
+     */
+    boolean bool = true;
+      // violation below, 'Block comment has incorrect indentation level 6'
+      /*
+       * (block comment should have the same indentation level as line 9)
```

Comment:
Please change content of comments to not have violation message. To clear define that is not a hint, it is just example comment.

Reply:
Done